### ISSUE TITLE
Attaching with console or qtconsole crashes kernel

### ISSUE THREAD

**Issue by fmagin** [Kotlin/kotlin-jupyter#133]
Using `jupyter-console --existing` or `jupyter-qtconsole --existing` to attach to the kernel crashes it. This bug is not yet part of the `kotlin-jupyter-kernel` package on PyPI, and only happens when building the kernel yourself or using the new published module from Maven.

I am using the kernel as a library, but I am fairly confident that this is an issue in regular usage too. 
The full stacktrace is:
```
kotlinx.serialization.json.internal.JsonDecodingException: Encountered unknown key 'raw'.
Use 'ignoreUnknownKeys = true' in 'Json {}' builder to ignore unknown keys.
Current input: {"raw":true,"output":false,"hist_access_type":"tail","n":1000}
	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24)
	at kotlinx.serialization.json.internal.JsonExceptionsKt.UnknownKeyException(JsonExceptions.kt:62)
	at kotlinx.serialization.json.internal.JsonTreeDecoder.endStructure(TreeJsonDecoder.kt:209)
	at org.jetbrains.kotlinx.jupyter.HistoryRequest$$serializer.deserialize(message_types.kt)
	at org.jetbrains.kotlinx.jupyter.HistoryRequest$$serializer.deserialize(message_types.kt:468)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:63)
	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:51)
	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readJson(TreeJsonDecoder.kt:24)
	at kotlinx.serialization.json.Json.decodeFromJsonElement(Json.kt:104)
	at org.jetbrains.kotlinx.jupyter.MessageDataSerializer.deserialize(message_types.kt:571)
	at org.jetbrains.kotlinx.jupyter.MessageDataSerializer.deserialize(message_types.kt:550)
	at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:63)
	at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:51)
	at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readJson(TreeJsonDecoder.kt:24)
	at kotlinx.serialization.json.Json.decodeFromJsonElement(Json.kt:104)
	at org.jetbrains.kotlinx.jupyter.ConnectionKt.receiveMessage(connection.kt:248)
	at org.jetbrains.kotlinx.jupyter.JupyterConnection$Socket.receiveMessage(connection.kt:64)
	at org.jetbrains.kotlinx.jupyter.IkotlinKt.kernelServer(ikotlin.kt:161)
	at org.jetbrains.kotlinx.jupyter.IkotlinKt.kernelServer$default(ikotlin.kt:97)

ZMQException - Errno 156384765 (SwingExceptionHandler) org.zeromq.ZMQException: Errno 156384765 : Context was terminated
	at org.zeromq.ZMQ$Socket.mayRaise(ZMQ.java:3546)
	at org.zeromq.ZMQ$Socket.recv(ZMQ.java:3377)
	at org.jetbrains.kotlinx.jupyter.IkotlinKt$kernelServer$$inlined$use$lambda$1.invoke(ikotlin.kt:148)
	at org.jetbrains.kotlinx.jupyter.IkotlinKt$kernelServer$$inlined$use$lambda$1.invoke(ikotlin.kt)
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)
```

**Comment by ileasile** [comment]
History request and history reply message formats are not implemented correctly:
https://jupyter-client.readthedocs.io/en/stable/messaging.html#history
It is needed to add implementation here:
https://github.com/Kotlin/kotlin-jupyter/blob/65b3a57e9bf8af0a1a0669e683c930a626bf9e77/src/main/kotlin/org/jetbrains/kotlinx/jupyter/message_types.kt#L468

**Comment by ileasile** [comment]
Should be fixed by #134
